tests/data_out/data_out_curved_cells.cc

```diff
@@ -158,7 +158,9 @@ curved_grid(std::ostream &out)
       for (const unsigned int vertex_no : GeometryInfo<2>::vertex_indices())
         {
           for (unsigned int i = 0; i < 2; ++i)
-            m[i].add_line(cell->vertex_dof_index(vertex_no, 0));
+            if (m[i].is_constrained(cell->vertex_dof_index(vertex_no, 0)) ==
+                false)
```

Comment:
I had started to write the test with the code as it was before, where it would have been an error to call the function twice. I then changed my mind, and this kind of thing here is the result.

I'm not feeling strongly about it either way. If you want me to, I can go back and remove this line. But I will also be very happy if we just leave it in, given that it's only in the tests, and use the time so saved for something more productive :-)